packages/components/src/card/style.scss

```diff
@@ -69,6 +69,10 @@
 	&.is-warning {
 		border-left: 3px solid var(--color-warning-20);
 	}
+
+	&.is-primary {
+		border-left: 3px solid var(--color-accent);
+	}
```

Comment:
Sorry if this is a random question! But does this not just make it equivalent to the `Banner` component? Might I be missing something about why that can't be used instead?

Reply:
It's a good question!

I tried using the banner component but I found that the icon was getting the accent background and border radius whenever I tried using the GitHub social logo.

I'll try again though; I prefer that approach.

Reply:
Ah, I see! You should be able to use the `disableCircle` prop for that. I haven't tested it but I think you could either add a `socialIcon` prop to the component here:

https://github.com/Automattic/wp-calypso/blob/035f94a42eb8d8e30450e6853ec6eef02385e692/client/components/banner/index.jsx#L196-L200

Or pass the GitHub logo as an image with the `iconPath` prop - seems like there's one lying around in Calypso, though not sure how ideal that is: `/calypso/images/sharing/p2-github-logo.svg` (might need to style the size down though)

Reply:
Yeah! What I'm doing is modify the banner component to accept a `ReactNode` as well. Thanks for pushing; I will be able to use the banner component indeed!